circle.yml

```diff
@@ -15,41 +15,6 @@ workflows:
   development:
     jobs:
       - sanity-checks
-      - build-template:
-          name: android-armeabi-v7a-release
-          executor_name: ubuntu-disco
-          target_is_android: true
-          requires:
-            - sanity-checks
-          config_params: '-G Ninja -DCMAKE_INSTALL_PREFIX=/tmp/workspace/install/$CIRCLE_JOB -DCMAKE_TOOLCHAIN_FILE=/opt/android/ndk-bundle/build/cmake/android.toolchain.cmake -DANDROID_CCACHE=/usr/bin/ccache -DANDROID_ABI=armeabi-v7a'
-          install: true
-      - build-template:
-          name: android-arm64-v8a-release
-          executor_name: ubuntu-disco
-          target_is_android: true
-          requires:
-            - sanity-checks
-          config_params: '-G Ninja -DCMAKE_INSTALL_PREFIX=/tmp/workspace/install/$CIRCLE_JOB -DCMAKE_TOOLCHAIN_FILE=/opt/android/ndk-bundle/build/cmake/android.toolchain.cmake -DANDROID_CCACHE=/usr/bin/ccache -DANDROID_ABI=arm64-v8a'
-          build_params: '--target libmapbox-gl.so'
```

Comment:
I'm concerned about this, we won't measure the binary size anymore if we do this. I'm testing a version of your PR here that keeps these android bots but will measure the size of the test runner at least.

Reply:
As agreed, we will bring this back and measure the size of the render-test-runner.so